docs/reference_guides/model_libraries/generic/costing/process_costing.rst

```diff
@@ -0,0 +1,865 @@
+
+.. contents:: Contents 
+    :depth: 4
+
+Introduction
+------------
+
+.. note:: The power plant costing method is available for most of the unit operations in the IDAES Unit Model Library (Compressor, CSTR, Flash, Heater, HeatExchanger, HeatExchangerNTU, PFR, PressureChanger, Pump, StoichiometricReactor, Turbine, and generic vessels).
+
+The process costing module outlines capital cost methodology for generic IDAES unit operations based on costing correlations that take unit configuration, material, size and other factors for unit costing into consideration.
+
+
+The framework includes factors for vessel shell thickness and dimensions, material properties for equipment shells and tubes, and tray types for distillation columns. The framework assumes a 2018 cost basis, and does not yet support heat exchangers requiring both shell and tube area (HX1D).
+
+When an instance of a unit model is passed to a `UnitModelCostingBlock`, a new sub-block is created 
+on that unit named `costing` (i.e. `flowsheet.unit.costing`). All variables and constraints related to costing will be 
+constructed within this new block (see detailed documentation for each unit for details on these variables and constraints).
+
+Note that a `FlowsheetCostingBlock` must already exist, and users must also pass the correct costing method from the SSLW module as an argument (see the example below for details on proper implementation).
+
+
+Table 1. Main Variables added to the unit block ("self.costing").
+
+=========================== ============================== ============ ===============================================================================
+Variable                    Symbol                         Units        Notes
+=========================== ============================== ============ ===============================================================================
+Purchase cost               :math:`purchase\_cost`         dollars      Purchase cost
+Base cost per unit          :math:`base\_cost\_per\_unit`  unitless     Base cost per unit
+Base cost                   :math:`base\_cost`             unitless     Base cost (base cost per unit * number of units)
+Number of units             :math:`number\_of\_units`      unitless     Number of units to be costed (to take advantage of the economics of scale)
+=========================== ============================== ============ ===============================================================================
+
+.. note:: number of units by default is fixed to 1 and the user must unfix this variable to optimize the number of units. Also, `number of units` can be built as a continuous variable or an integer variable. If the latter, the user must provide an mip solver. Use the global costing argument for this purpose (integer_n_units=True or False).
+
+Example
+^^^^^^^
+Below is a simple example of how to add cost correlations to a flowsheet including a heat exchanger using the default IDAES costing module.
```

Comment:
Do not call this the "default" costing package - name it after the reference (this is going back to my earlier comment about emphasizing the general framework over the specific costing packages).